Source/CBL_Router+Handlers.m

```diff
@@ -293,11 +296,17 @@ - (CBLStatus) do_POST_bulk_docs: (CBLDatabase*)db {
                 } else if (status >= 500) {
                     return status;  // abort the whole thing if something goes badly wrong
                 } else if (allOrNothing) {
+                    if (error)
+                        _response.statusReason = error.localizedFailureReason;
                     return status;  // all_or_nothing backs out if there's any error
                 } else {
-                    NSString* error = nil;
-                    status = CBLStatusToHTTPStatus(status, &error);
-                    result = $dict({@"id", docID}, {@"error", error}, {@"status", @(status)});
+                    NSString* errorMessage = nil;
+                    status = CBLStatusToHTTPStatus(status, &errorMessage);
+                    NSString* reason = error.localizedFailureReason;
+                    if (reason)
```

Comment:
You don't need to check `reason` — `$dict` ignores nil values (unlike `@{...}`.) So you can just use the first assignment line (307).